rpc/gnmi/mixed-schema.md

```diff
@@ -108,50 +109,187 @@ if all transactions succeed should the `SetResponse` indicate success. If any
 of the transactions fail, the contents of all origins MUST be rolled back, and
 an error status returned upon responding to the `Set` RPC.
 
-## Example
+## Special considerations for a setRequest with openconfig and cli origins
+
+### Use cases
+The operational use cases for a setRequest `replace` operation containing both OC and CLI origins include:
```

Comment:
As mentioned in the other PRs, we are talking about "mixed schema" here but keep concentrating on CLI only as the native encoding/style

Again this use-case should cover and reference "native" vs. strict CLI usage - this is driving a single use-case that I'm going to venture to bet that other folks that have adopted YANG modeling and other NBI (e.g. NETCONF) are _not_ sending CLI-syntax payloads to network-elements any longer.  The concentration on CLI here leaves a gap in the specification for the cases where we are truly talking about mixed schemas that could be encoded in JSON_IETF or XML already